pyproject.toml

```diff
@@ -207,3 +207,22 @@ showcontent = true
 directory = "test"
 name = "Test"
 showcontent = true
+
+[tool.bandit]
+exclude_dirs = ["tests"]
+skips = [
+    "B105",
+    "B110",
+    "B112",
+    "B307",
+    "B310",
+    "B311",
+    "B314",
+    "B403",
+    "B404",
+    "B405",
+    "B602",
+    "B603",
+    "B604",
+    "B607",
+    ]
```

Comment:
Hi @RobPasMue 

We have added a skip list of known Python limitations for Bandit but still it is showing errors for them [here](https://github.com/ansys/pyfluent/actions/runs/14708334375/job/41274091910?pr=3968).

We are primarily focusing on fixing 3rd party vulnerabilities first then we will focus on native Python ones.  

Please share your views.

Reply:
Hi @hpohekar! Could you give it a try to using a ".bandit" file rather than the pyproject.toml approach? https://bandit.readthedocs.io/en/latest/config.html

Reply:
@RobPasMue I used `.bandit` file still it's failing.

https://github.com/ansys/pyfluent/actions/runs/14730236191/job/41342853002

Reply:
Hi @RobPasMue 

I investigated it further, I recall that when I was using it offline then I was passing `c`, `pyproject.toml` as follows and it was working fine.

![image](https://github.com/user-attachments/assets/90c8d255-3c94-4eba-8bc3-eae8c3ac592d)

There is also another way to pass `args` and `additional_dependencies` as follows but same options are not allowed on our hook (ansys/actions/check-vulnerabilities@v9)

![image](https://github.com/user-attachments/assets/85b781f0-db06-4d66-99a4-cb7c0de13656)

Reply:
Okay -- this has to be supported through the actions. Can you open an issue pointing to this thread? I will have the team work on it

Reply:
@RobPasMue Done. Thank you.

https://github.com/ansys/actions/issues/793